cli/packaging/ubuntu/postinst

```diff
@@ -18,6 +18,11 @@ case "$1" in
     # This value is present only during upgrades
     if [ -z "$2" ]; then
         ln -s /usr/bin/splinter /usr/bin/splinter-cli
+
+        mkdir /usr/local/share/splinter/circuit-templates
```

Comment:
You should check if the directory exists first and skip this step if it does (same in the gameroom one), otherwise this fails.

Reply:
addressed